test/performance/io/format_fastq_benchmark.cpp

```diff
@@ -0,0 +1,285 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+// @author: Clemens Cords <clemens.cords@fu-berlin.de>
+
+#include <benchmark/benchmark.h>
+#include <cstring>
+#include <fstream>
+
+#define SEQAN2_HAS_SEQAN3 __has_include(<seqan/seq_io.h>)
+#if SEQAN2_HAS_SEQAN3
+    #include <seqan/seq_io.h>
+#endif
```

Comment:
We have now a header to check for `seqan2`

```c++
#include <seqan3/test/seqan2.hpp>

// defines macro SEQAN3_HAS_SEQAN2
```

Can you include this header and replace `SEQAN2_HAS_SEQAN3` with `SEQAN3_HAS_SEQAN2`

seqan3/test/seqan2.hpp:
https://github.com/seqan/seqan3/blob/063af063f535098adc9a1bcf75810dd3726c8bb5/test/include/seqan3/test/seqan2.hpp

Reply:
fixed